core/controllers/suggestion_test.py

```diff
@@ -1719,9 +1828,7 @@ def test_accept_question_suggestion_with_image_region_interactions(self):
                 suggestion.target_id,
                 suggestion.suggestion_id), {
                     'action': u'accept',
-                    'commit_message': u'commit message',
```

Comment:
Why is this field dropped?

Reply:
Because these fields are passed to `SuggestionToSkillActionHandler` and there we are not expecting these fields
https://github.com/oppia/oppia/blob/5471941da48071723b5f7e8684716584c08c6a1d/core/controllers/suggestion.py#L213